File: clients/kotlin/src/main/kotlin/ai_cup_22/MyStrategy.kt
class MyStrategy(constants: ai_cup_22.model.Constants) {
    fun getOrder(game: ai_cup_22.model.Game, debugInterface: DebugInterface?): ai_cup_22.model.Order {
        return ai_cup_22.model.Order(mutableMapOf())
    }
    fun debugUpdate(debugInterface: DebugInterface?) {}
    fun finish() {}
}

File: clients/scala/src/main/scala/ai_cup_22/MyStrategy.scala
class MyStrategy(constants: ai_cup_22.model.Constants) {
  def getOrder(game: ai_cup_22.model.Game, debugInterface: Option[DebugInterface]): ai_cup_22.model.Order =
    ai_cup_22.model.Order(Map.empty)
  def debugUpdate(debugInterface: DebugInterface): scala.Unit = ()
  def finish(): scala.Unit = ()
}